### ISSUE TITLE
Unable to connect to search provider Hounddawgs

### ISSUE THREAD

**Issue by mattail** [SickRage/SickRage#3578]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/master
Commit: a1f5e6750493b00b032536c9a4274da57b5b9531
OS: Synology / Linux-3.10.35-x86_64-with-glibc2.2.5 
What you did: While adding a new TV-show I noticed that the show didn't showed up in the download queue. Looking at the log view I noticed these lines:
What happened:
What you expected:
Logs:
```
PASTE LOGS HERE

2017-04-02 16:50:35 WARNING SEARCHQUEUE-MANUAL-279536 :: [HoundDawgs] :: Unable to connect to provider
2017-04-02 16:50:35 INFO SEARCHQUEUE-MANUAL-279536 :: [HoundDawgs] :: Request failed: ('Connection aborted.', BadStatusLine("''",))
2017-04-02 16:50:35 DEBUG SEARCHQUEUE-MANUAL-279536 :: [HoundDawgs] :: GET URL: https://hounddawgs.org/ [Status: 200]
2017-04-02 16:50:34 INFO SEARCHQUEUE-MANUAL-279536 :: [HoundDawgs] :: Performing episode search for Billions
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [x] Post debug logs, either inline (for smaller logs) or using gist
- [x] Please wrap inline logs inside \`\`\` and \`\`\` for readability

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by mattail** [comment]
I digged i bit more in the debug logs and found out that I haven't been able to get any torrents from Hounddawgs since the update to commit  "a1f5e6750493b00b032536c9a4274da57b5b9531" on the 2017-03-31. I have a 2nd search provider that's why I didn't noticed this earlier.

**Comment by mattail** [comment]
Looks like problem is solved.
I stopped sickrage from synology package center.
SSH to sickbeard-custom directory and updated manully with git commands.
Started sickrage again from synology package center.

To test, I removed my 2nd search provider, deleted a random episode and manually search for that episode. Now the episode was found, also when inspecting the logs I see that I'm able to connect Hounddawgs again.
